doc/fd.1

```diff
@@ -32,7 +32,7 @@ Include hidden files and directories in the search results
 .TP
 .B \-I, \-\-no\-ignore
 Show search results from files and directories that would otherwise be ignored by
-.IR .gitignore ,
+any of the files ignored by the '\-\-no\-ignore\-vcs' option,
```

Comment:
I think I understand your intention here (avoid duplication), but since the `-no-ignore` option is much more prominent than `--no-ignore-vcs`, I would could maybe phrase this a bit differently. Or maybe make it more clear by showing it in a list (I had to read the sentence twice until I understood it - but that might be just me).

```
Show search results from files and directories that would otherwise be ignored by
  - .gitignore (and related Git ignore files, see --no-ignore-vcs)
  - .ignore
  - .fdignore
  - the global ignore file
```

I think we could also add: "the global fd ignore file"


What do you think?

Reply:
sure, I can do that.